src/Ng/CommandHelpers.cs

```diff
@@ -97,13 +95,16 @@ public static void AssertAzureStorage(IDictionary<string, string> arguments)
             }
         }
 
-        public static RegistrationStorageFactories CreateRegistrationStorageFactories(IDictionary<string, string> arguments, bool verbose)
+        public static RegistrationStorageFactories CreateRegistrationStorageFactories(
+            IDictionary<string, string> arguments,
+            bool verbose,
+            IThrottle throttle = null)
         {
             CatalogStorageFactory legacyStorageFactory;
-            var semVer2StorageFactory = CreateSemVer2StorageFactory(arguments, verbose);
+            var semVer2StorageFactory = CreateSemVer2StorageFactory(arguments, verbose, throttle);
```

Comment:
you should merge with latest dev..

Reply:
I did. May be a left-over of an earlier throttling implementation on this branch by @scottbommarito before the change went into `dev` via separate PR. See also my comment [here](https://github.com/NuGet/NuGet.Services.Metadata/pull/524/files#r285221099).

Reply:
Fixed!